### ISSUE TITLE
Initial namespace backslashes are marked as errors in PHP

### ISSUE THREAD

**Issue by Racso** [rouge-ruby/rouge#1977]
**Name of the lexer**
PHP

**Code sample**
```
function myFunction(\MyApp\MyClass $param): void
{
    $param->doSomething(\App\MyOtherClass::class);
}
```

**Result**

![image](https://github.com/rouge-ruby/rouge/assets/26185386/0ddc4295-8e3a-4ac7-b286-78b673486408)

- Expected result: All the code is parsed correctly.
- Actual result: Both initial backslashes `\` are marked as errors. They are valid PHP.

**Additional context**
- I'm using GitHub pages.

**Comment by nsfisis** [comment]
This issue appears to be the one I previously addressed. The fix is included in version 4.1.1 or later.

[CHANGELOG](https://github.com/rouge-ruby/rouge/blob/1687d63cede01e9e1c108425e9987060ad85c79d/CHANGELOG.md#version-411-2023-05-15)

> php: fix highlight of fully-qualified identifiers (fix #1718) (https://github.com/rouge-ruby/rouge/pull/1924/ by nsfisis)

**Comment by tancnle** [comment]
Closed since it has already been fixed.
